compiler-core/src/error.rs

```diff
@@ -1032,20 +1032,67 @@ https://gleam.run/getting-started/installing/",
 Documentation for installing rebar3 can be viewed here:
 https://gleam.run/getting-started/installing/",
                     ),
-                    _ => (),
-                }
-                match (program.as_str(), env::consts::OS) {
-                    // TODO: Further suggestions for other OSes?
-                    ("erl" | "erlc" | "escript", "macos") => text.push_str(
+                    "bun" => text.push_str(
+                        "
+Documentation for installing bun can be viewed here:
+https://bun.sh/docs/installation/
+You may need to restart your shell after installing bun.",
+                    ),
+                    "deno" => text.push_str(
                         "
-You can also install Erlang via homebrew using \"brew install erlang\"",
+Documentation for installing Deno can be viewed here:
+https://docs.deno.com/runtime/getting_started/installation/",
                     ),
-                    ("rebar3", "macos") => text.push_str(
+                    "node" => text.push_str(
                         "
-You can also install rebar3 via homebrew using \"brew install rebar3\"",
+Documentation for installing Node.js via package manager can be viewed here:
+https://nodejs.org/en/download/package-manager/all/",
                     ),
                     _ => (),
-                };
+                }
+                match env::consts::OS {
+                    "macos" => {
+                        fn brew_install(program: &str) -> String {
+                            format!("\nYou can install {} via homebrew: brew install {}", program, program)
+                        }
+                        match program.as_str() {
+                            "erl" | "erlc" | "escript" => text.push_str(&brew_install("erlang")),
+                            "rebar3" => text.push_str(&brew_install("rebar3")),
+                            "deno" => text.push_str(&brew_install("deno")),
+                            "elixir" => text.push_str(&brew_install("elixir")),
+                            other => text.push_str(&wrap_format!("
+You might need to install {other} manually on your macOS."
+                            )),
+                        }
+                    }
+                    "linux" => {
+                        fn apt_install(program: &str) -> String {
+                            format!("\nYou can install {} via apt: sudo apt install {}", program, program)
```

Comment:
This isn't using the right name for the packages, and the package in Debian and Ubuntu stable is not something we want to recommend due to the maintainers of the repository using an old version and having broken it by removing parts of OTP for some reason.

Reply:
seems like using apt to install erlang-* is not good and I removed it